Gemfile

```diff
@@ -8,7 +8,7 @@ git_source(:github) do |repo_name|
 end
 
 gem 'sprockets'
-gem 'elastic-apm'
+gem 'elastic-apm', '4.5.0'
```

Comment:
https://github.com/elastic/apm-integration-testing/tree/main/docker/opbeans/ruby could be the reason 

IIUC, the opbeans ruby for the apm-its will fetch the agent in different ways and then spin up this docker image, will forcing the gem version in the Gemfile cause any issues in the above consumer?

Reply:
AFAIK it should not have an impact since `bundler` is used as a gem manager.
We should be able to install a specific version using the Gemfile and then install another version running the `gem` command.
It's worth to note that installing a dependency using `gem` command when another version is already installed means that we have both versions at the end.

```bash
# Using the above integration
docker build -t test .
docker run -it --rm -e RUBY_AGENT_VERSION=4.4.0 --entrypoint=/bin/sh test

$ gem list | grep "elastic-apm"
> elastic-apm (4.5.0)

$ ./entrypoint.sh
$ gem list | grep "elastic-apm"
> elastic-apm (4.5.0, 4.4.0)
```

Reply:
> It's worth to note that installing a dependency using gem command when another version is already installed means that we have both versions at the end.

IIUC, then this behaviour was already in place anyway, so this change should not affect by any means. Thanks for double checking it 

Out of curiosity, which one has higher precedence?

Reply:
Bundler has the highest precedence, it basically locks out gems other than what are specified in the Gemfile.lock file